test/e2e/util/util.go

```diff
@@ -64,6 +64,26 @@ func GetTunedForNode(cs *framework.ClientSet, node *corev1.Node) (*corev1.Pod, e
 	return &podList.Items[0], nil
 }
 
+// GetTunedForNode returns a Pod that runs on a given node.
+func GetNodeTuningOperatorPod(cs *framework.ClientSet) (*corev1.Pod, error) {
+	listOptions := metav1.ListOptions{
+		LabelSelector: labels.SelectorFromSet(labels.Set{"name": "cluster-node-tuning-operator"}).String(),
+	}
+
+	podList, err := cs.Pods(ntoconfig.OperatorNamespace()).List(context.TODO(), listOptions)
+	if err != nil {
+		return nil, fmt.Errorf("couldn't list potential NTO operator pods: %v", err)
+	}
+
+	if len(podList.Items) != 1 {
+		if len(podList.Items) == 0 {
+			return nil, fmt.Errorf("failed to find the cluster-node-tunig-operator pods")
+		}
+		return nil, fmt.Errorf("too many (%d) cluster-node-tunig-operator pods", len(podList.Items))
```

Comment:
Do we need to do this?  Say we have two operators running and one waiting to be elected as the leader.  Is this what are you worried about?  Wouldn't it be better to return just the first one?  For what we currently need this would be likely be better and wouldn't fail an e2e test if two operators were running at the same time.

Reply:
Yes, this is what I was trying to handle. Taking the first Pod is probably better... though if there are multiple running pods I'm not sure which one the metrics service might point to.

Reply:
Aaah, good point.  I should probably leave the reviews for the morning. :)